docs/publish-docs.sh

```diff
@@ -30,6 +30,25 @@ cat > "$CONFIG_FILE" <<EOF
   "name": "$PROJECT_NAME",
   "scope": "solana-labs"
 }
+{
+  "redirects": [
```

Comment:
@danpaul000 Can you look over these, is there any way to test this before they are released?